src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -228,5 +229,17 @@ public static class Task
                 public static readonly string DisplayName = "task.displayname";
             }
         }
+
+        public class Release
+        {
+            public class Path
+            {
+                public static readonly string WorkingDirectory = "r";
```

Comment:
To add some more context on this. It appears what you're adding is a bit of a misnomer. It's another level in the "_work" directory (not a "working" directory).

The _work folder is used for things today that is shared across Build and Release (e.g. Node handler creates some temp files there). Also there is plans to add a System.TempDirectory under the "_work" folder. I would rather not introduce another level for release, and add confusion which folder is *the* "_work" folder.